fili-core/src/test/groovy/com/yahoo/bard/webservice/web/endpoints/DruidQueryIdSpec.groovy

```diff
@@ -0,0 +1,80 @@
+// Copyright 2016 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.web.endpoints
+
+import com.yahoo.bard.webservice.data.dimension.BardDimensionField
+import com.yahoo.bard.webservice.data.dimension.DimensionDictionary
+import com.yahoo.bard.webservice.util.JsonSlurper
+import com.yahoo.bard.webservice.util.JsonSortStrategy
+import com.yahoo.bard.webservice.web.filters.BardLoggingFilter
+
+import javax.ws.rs.core.MultivaluedHashMap
+
+
+class DruidQueryIdSpec extends BaseDataServletComponentSpec {
```

Comment:
It would be good to have this name reflect something about looking for the request ID header to be connected to the druid query id. Perhaps something like `RequestIdPrefixesDruidQueryIdSpec`?